packages/themes/tasks/docs.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright IBM Corp. 2015, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const { createMarkdownFile } = require('@carbon/build-utils/sassdoc');
+
+createMarkdownFile('./scss', './docs/sass.md');
```

Comment:
On a top-level API note, the strategy for tasks tends to be:

```js
async function taskName() {
  // ...
}

taskName().catch(error => {
  console.error(error);
});
```

Would it be possible for `createMarkdownFile` to return a promise that we could include in this kind of block? e.g.

```js
async function taskName() {
  await createMarkdownFile('./scss', './docs/sass.md');
}

taskName().catch(error => {
  console.error(error);
});
```

Another thing that could be possible would be to have this be a task on `bundler` so that we could do something like `bundler docs scss` as a command in a build step.